Assets/MRTK/Examples/Demos/HandTracking/Prefabs/HandMenu_Small_HideOnHandDrop.prefab

```diff
@@ -151,7 +151,7 @@ GameObject:
   - component: {fileID: 4650336336052702770}
   - component: {fileID: 4650336336052702773}
   m_Layer: 0
-  m_Name: PalmUpHandMenu
```

Comment:
To clarify, since it looks like this prefab is being updated and renamed without the GUID changing, if I was previously using `PalmUpHandMenu.prefab` in my scene, will I see a behavior change when I update to MRTK 2.4 with this new prefab, or do they behave identically by default?

Reply:
By default the added functionality is set to false, so it will have the same functionality

Reply:
Yes, it will be identical.
![2020-04-20 11_05_21-Unity 2018 4 21f1 Personal -  PREVIEW PACKAGES IN USE  - HandMenuExamples unity ](https://user-images.githubusercontent.com/13754172/79784329-ee4fe300-82f6-11ea-8734-5b6f46f14ad4.png)

Reply:
It looks like `useGazeActivation` is set to true for this prefab: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7679/files#diff-98a1886079d65361204ca692bd06d7cfR271

Reply:
Just pushed a fix

Reply:
Or thought I did; have to merge

Reply:
Done for real this time